vllm/entrypoints/llm.py

```diff
@@ -26,6 +26,8 @@
 from vllm.usage.usage_lib import UsageContext
 from vllm.utils import Counter, deprecate_kwargs, is_list_of
 
+import numpy as np
```

Comment:
It's best not to import all names from a module, but just those names that you actually use. In other words, you should try to always use the `from ... import ...` form of imports.

Also, this import line probably should go with other "system"/"thirdparty" imports before the vllm-specific imports.

Reply:
please take a look